ansible/roles/iroha-docker/templates/docker-compose.yml.j2

```diff
@@ -29,7 +29,7 @@ services:
 {% endfor %}
 {% endif %}
     volumes:
-      - iroha_block_store-{{ node.human_hostname }}:/tmp/block_store
+      - {{ iroha_volume_dir }}iroha_block_store-{{ node.human_hostname }}:/tmp/block_store
```

Comment:
You maybe have to use conditional and "path" filter.

Reply:
There are no `path` filter I used `dirname`, and put it to top of file